src/utils/misc.h

```diff
@@ -59,6 +60,7 @@ namespace Utils {
         bool downloadUrlToFile(QUrl url, QFile *file);
         QByteArray downloadUrl(QUrl url);
         QString genericCSS();
+        QMap<QString , QString> getSearchEnginesMap();
```

Comment:
code styling

Reply:
Can you specify what is the problem in this case ?

Reply:
space before `,`